src/num.cpp

```diff
@@ -23,7 +23,7 @@ namespace vrv {
 // Num
 //----------------------------------------------------------------------------
 
-Num::Num() : TextElement("num-"), AttLabelled()
```

Comment:
Removing label on num will break auto pagination. Why not suggesting to add this to MEI?

Reply:
Sorry, I did not realized it is there through TextElement.